pennylane_lightning/src/util/Util.hpp

```diff
@@ -36,6 +36,13 @@
 #define USE_CBLAS 1
 #else
 #define USE_CBLAS 0
+
+#define CblasTrans 0
+#define CblasNoTrans 0
+#define CblasRowMajor 0
+
+#define DOTU_STD_CROSSOVER (1 << 10)
+#define DOTC_STD_CROSSOVER (1 << 10)
```

Comment:
These may also be better used as [template non-type arguments](https://www.ibm.com/docs/en/zos/2.1.0?topic=arguments-template-non-type) in the functions they are used in. This way we can conditional compile it using templating, and not be required to specify it as a defined magic value.